### PR TITLE
Create bus.json

### PR DESCRIPTION
**Pull request by kjonosm**
- add wikidata
- deduplicate Brescia Mobilità / Brescia Mobilità SpA

### CODE REVIEW

**Comment by bhousel** [comment]
Thanks @kjonosm 👍 
I think we were editing this file at the same time, but I'll just accept your version and put my changes back in later..
(it was just removing "Campus")

File: data/transit/route/bus.json [hunk 1]
```diff
@@ -1270,11 +1270,11 @@
     },
     {
       "displayName": "BJBUS",
-      "id": "bjbus-a45453",
+      "id": "bjbus-a1f154",
       "locationSet": {"include": ["cn"]},
       "tags": {
         "network": "BJBUS",
-        "network:wikidata": "Q4881181",
+        "network": "Q4881181",
```
Comment (LaoshuBaby) [inline data/transit/route/bus.json:1277]: Hello, my friend, why delete the ```:wikidata``` ? There will be 2 same ```network``` key but have different values

Comment (bhousel) [reply data/transit/route/bus.json:1277]: Yes I think this must have been an accident - when I ran the `npm run build` it caught the error, so I fixed it in 03bbcf60a

**Comment by LaoshuBaby** [comment]
Thanks for your quickly action!


o(*￣▽￣*)ブ





------------------&nbsp;Original&nbsp;------------------
From: "Bryan ***@***.***&gt;; 
Date: 2021年3月21日(星期天) 晚上11:04
To: ***@***.***&gt;; 
Cc: ***@***.***&gt;; ***@***.***&gt;; 
Subject: Re: [osmlab/name-suggestion-index] Create bus.json (#4986)





 
@bhousel commented on this pull request.
 
 
In data/transit/route/bus.json:
 &gt;        "locationSet": {"include": ["cn"]},        "tags": {          "network": "BJBUS", -        "network:wikidata": "Q4881181", +        "network": "Q4881181",  
Yes I think this must have been an accident - when I ran the npm run build it caught the error, so I fixed it in 03bbcf6

**Pull request merged**